src/test/java/IMDbTop5MoviesTest.java

```diff
@@ -0,0 +1,29 @@
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import static com.codeborne.selenide.Selenide.*;
+
+public class IMDbTop5MoviesTest {
+
+    @Test
+    public void verifyTop5MoviesAndRatings() {
+        open("https://www.imdb.com/chart/top/?ref_=nv_mv_250");
+        ElementsCollection listOfMovies = $$(".ipc-metadata-list-summary-item");
+
+        int count = 0;
+        for (SelenideElement movie : listOfMovies) {
```

Comment:
I see you copied my code and this is fine, only thing that I am worried about is that you may not understand entirely what Is happening...

So, let's break it into pieces.

1. We open browser with the link to the direct page.
2. Then we get the list of movies (and their data)
3. Later in the for loop from the list we get each movie from list as separate element SelenideElement movie : listOfMovies
4. As we're looking for first 5 movies we limit it to 5 to break after it has been reached
5. Then we take this single movie element movie (with all the titles, year etc...)
6. Later we detect other elements by using String movieTitle = movie. and the element we want like title
7. and so on...